pom.xml

```diff
@@ -9,13 +9,6 @@
   <version>2.23.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <distributionManagement>
-    <repository>
-      <id>${artifactory_id}</id>
-      <url>${artifactory_url}</url>
-    </repository>
-  </distributionManagement>
-
   <repositories>
```

Comment:
Do we still need the repositories section? I think we can remove it since there is the `pluginRepository` one. I'll try on my machine, I'll keep you updated on this one.

Reply:
Nope, as [StackOverflow](https://stackoverflow.com/a/14396275/9306378) says you can't. Unlucky :confused: